_data/applications.yml

```diff
@@ -0,0 +1,175 @@
+# Entries on the page show up in the order listed here, but with
+# official: true (official) first
+# official: nil (unofficial; non-team RH developed) second (this is omission of `official`),
+# official: false (third party) last
+
+# The Cockpit group is special, it inherits source, issues, license for the `source: cockpit` entries
+cockpit:
+  source: https://github.com/cockpit-project/cockpit/
+  issues: https://github.com/cockpit-project/cockpit/issues
+  license: LGPL 2.1
+  comment: |
+    This is the default template that "source: cockpit" pulls from, so source, issues, and license don't need to be filled out everywhere.
+
+machines:
+  title: Virtual Machines
+  source: https://github.com/cockpit-project/cockpit-machines
+  issues: https://github.com/cockpit-project/cockpit-machines/issues
+  package: cockpit-machines
+  official: true
+  license: LGPL 2.1
+  description: |
+    Create, run, and manage virtual machines in Cockpit.
+
+podman:
+  title: Podman Containers
+  source: https://github.com/cockpit-project/cockpit-podman
+  issues: https://github.com/cockpit-project/cockpit-podman/issues
+  official: true
+  license: LGPL 2.1
+  description: |
+    Download, use, and manage containers in your browser. (Podman replaces Docker.)
```

Comment:
Likewise, the "in your browser" is redundant. Or it should at least be "in Cockpit" for consistency, depending on the resolution of the previous item.

Reply:
Yeah, both should be in your browser, probably. They're things that one usually doesn't do in a browser, so it's notable. I do agree that as they're on the Cockpit website and are called cockpit-foo as packages, it's redundant to say Cockpit. But browser is still notable.